docs/source/topics/backends/snmp.rst

```diff
@@ -0,0 +1,149 @@
+| resources  | - load: array containing average load on the cpu in the past minute, 5 minutes and 15 minutes respectively |
+|            | - memory:                                                                                                  |
+|            |   - total: total memory in bytes                                                                           |
+|            |   - buffered: buffered memory in bytes                                                                     |
+|            |   - free: free memory in bytes                                                                             |
+|            |   - cached: cached memory in bytes                                                                         |
+|            | - swap:                                                                                                    |
+|            |   - total: total swap storage in bytes                                                                     |
+|            |   - free: free swap storage in bytes                                                                       |
++------------+------------------------------------------------------------------------------------------------------------+
+| interfaces | Each interface is listed with the following information:                                                   |
+|            |                                                                                                            |
+|            | - name                                                                                                     |
+|            | - type                                                                                                     |
+|            | - statistics:                                                                                              |
+|            |                                                                                                            |
+|            |  - rx_bytes                                                                                                |
+|            |  - tx_bytes                                                                                                |
++------------+------------------------------------------------------------------------------------------------------------+
+
+AirOS example
+=============
+
+::
+
+ from netengine.backends.snmp import AirOS
+ device = AirOS("10.40.0.130")
+ device.name
+ 'RM5PomeziaSNode'
+ device.uptime_tuple
+ (121, 0, 5)  # a tuple containing device uptime hours, mins and seconds
+
+We have just called two simple properties on **device**, but we can ask **device** for more specific values or portions of the SNMP tree not included in the API, just type::
+ device.next("1.3.6")
+
+Otherwise, if you want simply a value of the tree just type::
+ device.get_value("oid_you_want_to_ask_for")
```

Comment:
these methods are not documented above so why are we showing them in the example?

I guess we can document `next` and `get_value`, but I would avoid documenting `uptime_tuple` because it would not be consistent with advising to use `to_netjson`.